src/pages/user/User.tsx

```diff
@@ -65,7 +65,7 @@ const User: React.FC = () => {
           {user.authority ? (
             <>
               <p>{t('authority.connected_info')}</p>
-              <MuiLink href={user.authority.identifiersMap.handle[0]}>{t('authority.see_profile')}</MuiLink>
+              <MuiLink href={user.authority.identifiersMap?.handle[0]}>{t('authority.see_profile')}</MuiLink>
```

Comment:
egentlig litt quick-fix, da feilen oppstår som en følge av at request for å hente authority feiler (cors?), og vi får data som ikke matcher Authority-typen for authority-typen. Fant ikke ut hvorfor TS ikke håndterer dette... :/